core/src/main/resources/reference.conf

```diff
@@ -328,8 +328,8 @@ filesystems {
       # Class to instantiate and propagate to all factories. Takes a single typesafe config argument
       class = "cromwell.filesystems.drs.DrsFileSystemConfig"
       config {
-        martha {
-          url = "https://martha-url-here"
+        resolver {
```

Comment:
Same as my other comment, just wondering if this should be `drsResolver` rather than `resolver`